docs/software/soroban-settings.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Soroban Settings
+---
+
+Soroban has a large set of settings stored on ledger than can be modified
+through a vlaidator vote. This documents describes how to propose a new settings
+upgrade, and how to examine a proposed upgrade. You can also look at the
+[commands.md](commands.md) doc for more details on the commands used below.
+
+## Propose a settings upgrade
+
+This section will describe how to propose a settings upgrade, but take a look at
+the [admin page](admin.md#upgrading-soroban-settings) for more information on
+how the settings upgrade mechanism works internally. **If you are just being asked
+to vote for an upgrade, please move on to the [Examine a proposed upgrade](#examine-a-proposed-upgrade) step for details on how to examine a
+proposed upgrade.**
+
+1. Using the stellar-xdr rust tool and a JSON file as input, run `stellar-xdr encode --type ConfigUpgradeSet JSON_FILE`. [phase1.json](../../soroban-settings/phase1.json) can be used as a template to propose upgrades.
```

Comment:
Should we add a pre-requisite step on how to install stellar-xdr tool?

Reply:
Good point. Updated